pages/governance.md

```diff
@@ -0,0 +1,281 @@
+surrender them.
+
+### Repository Maintainers
+
+[Repository maintainers](#repository-maintainers) have [Maintain][permissions]
+rights to an individual repository as an outside collaborator. Among other
+rights, this allows them to push code to a branch in the
+[JuMP-dev Github organization] instead of their personal fork, merge pull
+requests, and close issues.
+
+## Community Involvement
+
+The JuMP project highly values the contributions made by members of the
+community. As an open-source project, JuMP is both made for the community, and
+by the community.
+
+There are four main channels that JuMP uses to engage with the community.
+
+### Community Forum
+
+The community forum is a place for community members to post questions and
+receive help.
+
+The current forum is the "Optimization (Mathematical)" section of the
+[Julia Discourse Forum](https://discourse.julialang.org/c/domain/opt/13).
+
+### Developer Chatroom
+
+The developer chatroom is a chatroom for developer-focused discussions about
+JuMP.
+
+The current chatroom is the [JuMP-dev Gitter channel](https://gitter.im/JuliaOpt/JuMP-dev).
+
+### Monthly Developer Call
+
+The [Steering Council] hosts a monthly developer call to discuss JuMP-related
+business.
+
+The calls are currently scheduled on the fourth Thursday of every month at 14:00
+Eastern. For information on how to take part, join the [developer chatroom] and
+ask `@odow` or `@mlubin` for an invite to the monthly developer call.
+
+### JuMP-dev Workshops
+
+The [Steering Council] will periodically choose a Chair to organize a JuMP-dev
+workshop.
+
+Previous workshops include [Santiago, 2019](https://jump.dev/meetings/santiago2019/),
+[Bordeaux, 2018](https://jump.dev/meetings/bordeaux2018/), and [Boston, 2017](https://jump.dev/meetings/mit2017/).
+
+Announcements of future workshops will be communicated through the
+[community forum] and via the [jump.dev](https://jump.dev) website.
+
+## Transferring repositories to jump-dev
+
+The [JuMP-dev Github organization] exists to simplify the management of Github
+permissions on JuMP-related repositories. It is not a curation of all
+JuMP-compatible solvers and supporting packages.
+
+Instead, community members who develop new pure-Julia solvers or solver-wrappers
+should first add their package to the list of supported solvers in the
+[Installation Guide](https://jump.dev/JuMP.jl/stable/installation/#Getting-Solvers-1)
+of the JuMP documentation.
+
+Even once a package has matured, the bar for transferring repositories to the
+[JuMP-dev Github organization] is high, and can be summarized by the following:
+
+* The maintainance of the package would be simplified if [core contributors]
+  have [Admin permission][permissions] to the repository.
+
+Community members wishing to transfer a repository from their personal account
+to the [JuMP-dev Github organization] should contact the [core contributors]
+via the [developer chatroom].
```

Comment:
Maybe this is a good place to mention that we request you not use "JuMP" in the name of third-party packages.